bitnami/odoo/templates/deployment.yaml

```diff
@@ -203,6 +203,8 @@ spec:
           ports:
             - name: http
               containerPort: {{ coalesce .Values.containerPorts.http .Values.containerPort }}
+            - name: longpolling
+              containerPort: {{ coalesce .Values.containerPorts.longpolling .Values.containerPort }}
```

Comment:
I don't think this coalesce makes sense as the port must be different from the default port. I would do something like:

```
{{- if .Values.containerPorts.longpolling }}
- name: longpolling
  containerPort: {{ .Values.containerPorts.longpolling }}
{{- end }}
```